build-plugins-list.js

```diff
@@ -59,5 +61,5 @@ ${list}
 
 $3 END AUTO-GENERATED $4`
   );
-  fs.writeFileSync(file, newText);
+  fs.writeFileSync(file, prettier.format(newText, { parser }));
```

Comment:
Need to pass in correct prettier options here; I have a config in the package.json to use single quotes, etc. Probably should use https://prettier.io/docs/en/api.html#prettierresolveconfigfilepath-options to get the options.

Reply:
ah, I was searching what was the problem. I assumed the config was resolved automatically, but we're not in the #0cjs era yet :)

Reply:
Yeah, for some reason the API doesn't resolve the config.